README.md

```diff
@@ -1,2 +1,28 @@
 # Backgrounds-and-Gradients
-4rth Microverse Project 
+
+This project contains a clone of Apple website, with navigation bar styled with linear graduation in the background, main content with image as a background, content below it is a grid with some products available and then comes the footer with some final links.
+
+![](assets/project-screenshot.png)
+
+# Built with #
+ - HTML5 
+ - CSS3 
+ 
+# Live Demo #
+[Live Demo](https://rawcdn.githack.com/luciano-ilha/Backgrounds-and-Gradients/698dbbb34c0b639adb46d8f785bec9f02adffd4a/index.html)
+ 
+# Prerequisites #
+All Modern web browsers like:
+- Chrome 
+- Firefox
+- Safari
```

Comment:
Prerequisites should be for users who want to develop using your project, stating which browsers are required is unnecessary.

Reply:
Ok